Readme.md

```diff
@@ -183,365 +245,3 @@ After you get the records, from the transform tab convert data into a table. You
 
 ![](media/c809cf612f6589f202610125cb4d9fa9.png)
 
-# Connecting Power Apps and Power Automate to OSDU R2
```

Comment:
@VikramAtMicrosoft You should confirm with Kadri, if you haven't already, before you remove this section

Reply:
Okay will do, I have removed the section because the schema is outdated in the present code and the steps will not work, so we do not want to ship that. I have confirmed with my mentor Krishna Nikhil, I will also confirm with Kadri

Reply:
Vikram, what we can try is - Move Power Apps setup to different file say, PowerApps-Setup.md and move this documentation to that. 

Whenever PowerApps issues are sorted, we can use the documentation